test/empty-eownerdead.c

```diff
@@ -32,7 +32,10 @@ int main(int argc, char *argv[])
 	if (ret < 0) {
 		int __e = errno;
 
-		fprintf(stderr, "child: sqe submit failed: %s\n", strerror(__e));
+		if (__e == EOWNERDEAD)
+			fprintf(stderr, "sqe submit unexpected failure due old kernel bug: %s\n", strerror(__e));
+		else
+			fprintf(stderr, "sqe submit unexpected failure: %s\n", strerror(__e));
 		goto err;
 	}
```

Comment:
Ahh good, it should be more acceptable than 84875f42cab07d4b39d64b043cf9dbbdfe618674 ("test/empty-eownerdead: Only expect EOWNERDEAD as test success").

Reply:
> Ahh good, it should be more acceptable than 84875f42cab07d4b39d64b043cf9dbbdfe618674 ("test/empty-eownerdead: Only expect EOWNERDEAD as test success").

Oops, wrong reference.
6e72065e39e6bb7e744c6ba24d1a62a8d84d3e27 ("test/empty-eownerdead: Only expect EOWNERDEAD as test success").